mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/RowMatrix.scala

```diff
@@ -315,6 +315,30 @@ class RowMatrix(
   }
 
   /**
+    * Computes the inverse matrix using SVD
+    * @return a DenseMatrix
+    */
+  def computeInverse(): DenseMatrix = {
+  val nCoef = X.numCols.toInt
```

Comment:
The indentation and whitespace is off throughout this function, and I think one line won't pass style checks for its length. This is minor though.

The bigger concern I have is, where do we need an inverse versus solving a system? and the SVD is not always the best way to do it, etc. That is I don't know how much we go down this road of trying to expose another whole library for matrix ops.